.eslintignore

```diff
@@ -1,2 +1,3 @@
 package.json
 app/assets/javascripts/lib/
+app/javascript/storybook-static/
```

Comment:
Came across this when running the tests to ensure this folder was omitted in test coverage. It is omitted from coverage, but it wasn't from eslint.